video/vdu.h

```diff
@@ -22,8 +23,45 @@ void VDUStreamProcessor::vdu(uint8_t c) {
 	if (consoleMode) {
 		DBGSerial.write(c);
 	}
-	
+
+	if (printerOn) {
+		switch(c) {
+			case 0x03:
+				printerOn = false;
+				break;
+			// case 0x09:	// translate "cursor right" to a space, as terminals treat character 9 as tab
+			// 	vdu_print(32);
+			// 	break;
+			case 0x08 ... 0x13:
+			case 0x20 ... 0xFF: {
+				vdu_print(c);
+			}
+		}
+	}
+
+	if (!commandsEnabled) {
+		switch (c) {
+			case 0x01: {
+				// capture character and send to "printer" if enabled
+				auto b = readByte_t();	if (b == -1) return;
+				vdu_print(b);
+			}	break;
+			case 0x06:	// resume VDU command system
+				commandsEnabled = true;
+				break;
+		}
+		return;
+	}
```

Comment:
this version of handling `commandsEnabled` means we don't match up with Acorn's VDU system, as on an Acorn any commands sent would be effectively fetched in their entirety.

to match how Acorn's system worked, we'd need to embed checking for `commandsEnabled` into _every_ command.  this would be a _far_ more extensive change.

whilst what we have here won't be properly compatible, for the most part it should be "good enough"